tests/test_fts5_search.py

```
@@ -0,0 +1,339 @@
+    print("TESTING: FTS5 Search Functionality")
+
+    with tempfile.NamedTemporaryFile(suffix='.db', delete=False) as tmp:
+        tmp_path = Path(tmp.name)
+
+    try:
+        # Setup
+        db_manager = DatabaseManager(tmp_path)
+        db_manager.create_tables()
+
+        repo = SessionRepository()
+        search_engine = SmartSearchEngine()
```

Ruff (0.12.2)

Local variable `search_engine` is assigned to but never used

Remove assignment to unused variable `search_engine`

(F841)